src/frontend/src/banner.ts

```diff
@@ -14,7 +14,10 @@ export const showWarningIfNecessary = (): void => {
         href="https://github.com/dfinity/internet-identity#build-features"
         >more</a
       >`);
-  } else if (window.location.origin !== OFFICIAL_II_URL) {
+  } else if (
+    window.location.origin !== OFFICIAL_II_URL &&
```

Comment:
We should remove this banner for the new _and_ the old domain. We are actively going to publish documentation that points people to the new II domain as soon as it is live.

Let's treat them both as official here.